Makefile

```diff
@@ -42,19 +45,30 @@ bin/%:
 generate:
 	@hack/codegen.sh
 
-deploy: kind-load-manager
+deploy:
+	# We need to make sure the namespace is created before we apply any namespace-scoped configurations into cluster.
+	kubectl create namespace bulward-system
+	$(MAKE) deploy-manager
+	$(MAKE) deploy-apiserver
+
+deploy-manager: kind-load-manager
 	cd config/manager/manager && kustomize edit set image manager=${IMAGE_ORG}/bulward-manager:${VERSION}
 	kustomize build config/manager/default | kubectl apply -f -
 
+deploy-apiserver: kind-load-apiserver
+	cd config/apiserver/manager && kustomize edit set image manager=${IMAGE_ORG}/bulward-apiserver:${VERSION}
+	kustomize build config/apiserver/default | kubectl apply -f -
+	kubectl apply -f config/apiserver/rbac/extension_apiserver_auth_role_binding.yaml
+
 # ------------
 # Test Runners
 # ------------
 test:
-	go test -race -v ./...
+	CGO_ENABLED=1 go test -race -v ./...
 
-lint: generate pre-commit
+lint: pre-commit
 	@hack/validate-directory-clean.sh
-	golangci-lint run ./... --deadline=15m
+	golangci-lint run ./... --skip-files ".*generated.*" --deadline=15m
```

Comment:
`skip-files` skips auto-generated files to make the `make lint` happy

Reply:
Awesome!